share/spice/deb_version/deb_version.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_deb_version = function(api_result_str){
+        var api_result = JSON.parse(api_result_str);
+
+        if (api_result.error) {
```

Comment:
Check if `api_result` exists before accessing the `error` property, i.e.,

```javascript
if(!api_result || api_result.error) {
    ...
}
```

On that note, does the API actually return an `error` property when there's an error?